src/Psr3StdOutLogger.php

```diff
@@ -0,0 +1,37 @@
+<?php
+namespace Sil\Psr3Adapters;
+
+/**
+ * A basic PSR-3 compliant logger that writes logs to stdout. Note that this
+ * will bypass any output buffering that PHP may be doing.
```

Comment:
NOTE: this bypasses...
on its own line would be prettier.

Reply:
Done